FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -32,6 +31,14 @@ class MainConfig:
     except:
         _ip = "0.0.0.0"
 
+    # create the persistence path if it doesn't exist
+    if not os.path.exists(PERSISTENCE_PATH):
```

Comment:
We should also check that the path can be written to, using `os.access(PERSISTENCE_PATH, os.W_OK)`